test/OAuthClientTests.js

```diff
@@ -7,7 +7,6 @@ const app = express();
 const port = process.env.PORT || 3000;
 const host = process.env.HOST || 'localhost';
 /*
-const integratorKey = 'ae30ea4e-xxxx-xxxx-xxxx-fcb57d2dc4df'; // An IK for a non-mobile docusign account
 const integratorKeyImplicit = '68c1711f-xxxx-xxxx-xxxx-b49b4211d831'; // an IK with mobile selected in Docusign admin
 const clientSecret = 'b4dccdbe-xxxx-xxxx-xxxx-b2f0f7448f8f';
```

Comment:
same for clientSecret?

Reply:
@InbarGazit these appear to be scrubbed no?

Reply:
yes, but it's inconsistent with the others where you get them from env variables

Reply:
these are comments, they merely show the guid data type these are.

Reply:
Updated for consistency